cdn_lambda/functions/map_to_s3.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from chexus import Client, TableItem
+
+TABLE_NAME = "project-exodus-prototyping"
+TABLE_REGION = "us-east-2"
+
+
+def lambda_handler(event, context):
+    _ = context
+
+    # Get uri from origin request event
+    request = event["Records"][0]["cf"]["request"]
+
+    client = Client()
+
+    # Get DB table items with matching web_uri key
+    search_results = client.search(
+        item=TableItem(web_uri=request["uri"]),
+        table_name=TABLE_NAME,
+        region=TABLE_REGION
+    )
+
+    items = [TableItem(**i) for i in search_results["Items"]
+             if i.attrs["from_date"] <= datetime.utcnow().date()]
```

Comment:
Surprised we are only doing date here.

In scope of this issue we expect to work on DB schema a little, so this seems a reasonable place to point it out: I'm pretty sure these should be full ISO8601 timestamps down to the second, and not just dates.  Using only date makes it impossible to distinguish two publishes of the same piece of content within the same day, which is common.

Reply:
Ah, I hadn't considered multiple releases in a single day. Thanks.